resources/Materials/Examples/StandardSurface/user_bug.mtlx

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
```

Comment:
Could we have a comment (`<!-- comment -->`) that points out what the bug is?

Reply:
And I couldn't figure out where this file is used?

Reply:
hmm, I am actually not sure where that file is coming from... @muenstc  Do you know?